docs/version0.19.md

```diff
@@ -54,6 +55,10 @@ A new command line option [-XX:+PrintCodeCache](xxprintcodecache.md) allows you
 
 For Java 8 only, `StringBuffer` and `StringBuilder` grow differently when sized at 1 G `char[]` or larger. They immediately grow to the maximum possible size, similarly to Java 11 and later. Revert to the behavior of growing only as much as necessary to accommodate the `String` being added via the [`-Djava.lang.stringBuffer.growAggressively=false`](djavalangstringbuffergrowaggressively.md) option.
 
+### The default value of `-Xmx` has been changed in Java 8
+
+The default value of maximum heap size for Java 8 has been changed to match the behavior (25% of available memory with a maximum of 25 GB) on Java 11 or above. Revert to the old behavior (half the available memory with a minimum of 16MB and a maximum of 512 MB) in Java 8 via the [-XX:+OriginalJDK8HeapSizeCompatibilityMode](xxoriginaljadk8heapsizecompatibilitymode.md) option.
```

Comment:
Please include the Java 8 icon for this section, as shown in the previous one for StringBuffer. 

I'm not 100% OK with all the bracketed text in the description, so we might apply a little editing to this once it is in.